spec/presenters/openid_connect_user_info_presenter_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe OpenidConnectUserInfoPresenter do
+  include Rails.application.routes.url_helpers
+
+  let(:session_uuid) { SecureRandom.uuid }
+  let(:identity) { build(:identity, session_uuid: session_uuid, user: build(:user)) }
+
+  subject(:presenter) { OpenidConnectUserInfoPresenter.new(identity) }
+
+  describe '#user_info' do
+    subject(:user_info) { presenter.user_info }
+
+    it 'has basic attributes' do
+      aggregate_failures do
```

Comment:
Should we enable this globally (in a separate PR)?

https://www.relishapp.com/rspec/rspec-core/docs/expectation-framework-integration/aggregating-failures:
> You can also use config.define_derived_metadata to apply this to every example automatically.